user_guide/feature_contexts.rst

```diff
@@ -54,6 +55,22 @@ technical detail exposed in them, so the way you will test those
 features pretty much depends on the context you test them in. That's what
 context classes are.
 
+.. tip::
+
+  This class can be easily created by running
+  :doc:`Behat command line tool</user_guide/command_line_tool>` with the
```

Comment:
Replaced with
`The Behat command line tool has an '--init' option that will initialize a new Behat project in your
   directory. Learn more about 'initialize'`